Snippets/backend/connect/view/main.ini

```diff
@@ -789,7 +789,7 @@ import/not_allowed_drag_local_products = "Drag-and-drop nur bei importierten Pro
 info/new_version_header = "Neue Shopware Connect Version verfügbar."
 info/new_version_text = "Es ist eine neue Shopware Connect Version verfügbar. Bitte aktualisieren Sie das Plugin über den Plugin-Manager."
 plugin/update/header = "Neues Connect Plugin-Update"
-plugin/update/text = "Ein neues Update für das Connect Plugin ist verfügbar, bitte Update es, um alle Funktionen zu nutzen"
+plugin/update/text = "Ein neues Update für das Connect Plugin ist verfügbar. Bitte installiere die aktuellste Version, um alle neuen Funktionen und Optimierungen nutzen zu können."
```

Comment:
please update the english translation also to something like:
A new Connect plugin update is available. Pleaseinstall the latest version to use new functions and improvements